app/views/organizations/pets/index.html.erb

```diff
@@ -14,12 +14,21 @@
   <div class="container mb-md-5 mt-md-5">
 
     <!--page heading-->
-    <div class="text-center mb-5">
-      <h1 class="section-heading text-uppercase underline">
-        <%= t('.our_pets') %>
-      </h1>
-    </div>
+    <%= content_for :header do  %>
+      <div class="text-center mb-5">
+        <h2 class="section-heading text-uppercase">
+          <%= t('.our_pets') %>
+        </h2>
+      </div>
+    <% end %>
 
+    <!--create pet button-->
+      <%= content_for :button do  %>
+        <div class ="col-md-6 d-flex justify-content-center align-items-center">
+          <%= button_to t('.create_pet'), new_pet_path, class: "btn btn-primary" %><br>
+        </div>
```

Comment:
```suggestion
          <%= button_to t('.create_pet'), new_pet_path, class: "btn btn-primary" %><br>
```
I think you can change it to this since the `col-md-6` is shrinking the button and I think it will look fine without the surrounding `div`